tests/ZendTest/Session/Service/SessionManagerFactoryTest.php

```diff
@@ -85,6 +88,8 @@ public function testFactoryWillAddValidatorViaConfiguration()
         $this->services->setService('Config', $config);
         $manager = $this->services->get('Zend\Session\ManagerInterface');
 
+        $manager->start();
```

Comment:
This changes the context for the test! The original test omitted it for a specific reason, based on the security issue we addressed.